de/importData.py

```diff
@@ -11,6 +11,8 @@ def importLINCS(cellLine):
     data = np.load(join(path_here, "de/data/", cellLine + "_RNAi_matrix.npy"))
     annotation = pd.read_csv(join(path_here, "de/data/", cellLine + "_genes.txt"), header=None)
 
+    if data.shape[0] != data.shape[1]:
```

Comment:
You shouldn't have to do this, so long as the control is the last column. In that case it should be handled correctly.